mne/preprocessing/ica.py

```diff
@@ -120,7 +120,24 @@ class ICA(ContainsMixin):
     u"""M/EEG signal decomposition using Independent Component Analysis (ICA).
 
     This object can be used to estimate ICA components and then remove some
-    from Raw or Epochs for data exploration or artifact correction.
+    from Raw, Epochs or Evoked for data exploration or artifact correction.
+
+    Prior to fitting and applying the ICA, data is whitened (de-correlated and
+    scaled to unit variance, also called sphering transformation) by means of
+    a Principle Component Analysis (PCA). In addition to the whitening, this
+    step introduces the option to reduce the dimensionality of the data, both,
+    prior to fitting the ICA and prior to reverting to sensor space,
+    controllable by the two parameters `max_pca_components` and
+    `n_pca_components`, respectively.
+    .. note:: Commonly used for reasons of i) computational efficiency and
```

Comment:
you need a newline before this `.. note::` directive, see:

https://9997-1301584-gh.circle-artifacts.com/0/html/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA